policy/PfsMapper.yaml

```diff
@@ -385,6 +385,15 @@ datasets:
     tables:
     - raw
 
+  dmQaResidualImage_pickle:
```

Comment:
Is this an image that could be written as FITS?

Reply:
It's a `QaDict` with stats info, so it could be a numpy table in a FITS file but I was trying to follow the extraction example, which includes the word `Image` in the name because it is outputting to the same folder as the plot (I'm guessing).

Reply:
Let's try to get the word "stats" in there.